arkouda/array_api/_manipulation_functions.py

```diff
@@ -14,26 +14,67 @@ def broadcast_arrays(*arrays: Array) -> List[Array]:
 
     See its docstring for more information.
     """
-    raise ValueError("broadcast_arrays not implemented")
+
+    shapes = [a.shape for a in arrays]
+    bcShape = shapes[0]
+    for shape in shapes[1:]:
```

Comment:
Do we need a check here to see if `shapes` is of length 1 or is `[1:]` just `None` if it is length 1? Or does this function require more than 1 array?

Reply:
`shapes[1:]` returns an empty list if `shapes` only has a length of 1. So I believe this should essentially just create a copy of the array if it's only called on one array.

> Or does this function require more than 1 array?

And it looks like the [spec](https://data-apis.org/array-api/latest/API_specification/generated/array_api.broadcast_arrays.html#array_api.broadcast_arrays) says "1 or more arrays", so I don't think an extra check would be needed.